test/docs.jl

```diff
@@ -22,73 +22,6 @@ end
 @test (@doc ModuleMacroDoc)    == "I am a module"
 @test (@doc ModuleMacroDoc.@m) == ["I am a macro"]
 
-# apropos function testing
-
-@test sprint(apropos, "non-documented object") == "No help information found.\n"
```

Comment:
Looks like there are still vestiges of it at
```
base/docs/basedocs.jl:20:  You can also use `apropos("...")` to search the documentation.
base/exports.jl:1077:    apropos,
contrib/BBEditTextWrangler-julia.plist:192:              <string>apropos</string>
doc/stdlib/base.rst:144:.. function:: apropos(string)
```
which should be deleted if it's being completely removed